flow/Platform.actor.cpp

```diff
@@ -2424,23 +2424,7 @@ static bool deletedFile() {
 }
 
 bool deleteFile(std::filesystem::path path) {
```

Comment:
also here const reference, you are copying.

Reply:
fixed!